src/ProjectTemplates/Web.ProjectTemplates/GrpcService-CSharp.csproj.in

```diff
@@ -12,6 +12,7 @@
 
   <ItemGroup>
     <PackageReference Include="Grpc.AspNetCore" Version="${GrpcAspNetCoreVersion}" />
+    <PackageReference Include="Google.Protobuf" Version="3.22.0-rc2" />
```

Comment:
Why isn't this transitive?

Reply:
This is here to override the transitive dependency. The comment on the PR explains why it is here.

Reply:
File an issue to remove it later?

Reply:
https://github.com/dotnet/aspnetcore/issues/46465